tweaks.sh

```diff
@@ -67,7 +67,13 @@ while [[ $# -gt 0 ]]; do
           edit_firefox="true" ;;
       esac
 
-      if ! has_command firefox; then
+      if [[ -d "${FIREFOX_FLATPAK_DIR_HOME}" ]]; then
+          FIREFOX_DIR_HOME=${FIREFOX_FLATPAK_DIR_HOME}
```

Comment:
Thanks for your PR, but, it's a bad practice to change the environment variable like this

Reply:
Thanks for your feedback. Do you have a better idea to solve it?

Reply:
Just keep it there :D